opm/input/eclipse/Units/Units.hpp

```diff
@@ -321,6 +322,7 @@ namespace Opm {
         constexpr const double Moles                = kilo*pound*mol;
         constexpr const double PPM                  = ppm;
         constexpr const double Ymodule              = giga*Pascal;
+        constexpr const double ThermalConductivity  = btu/(degFahrenheit*day*centi*meter);
```

Comment:
I would normally expect `feet` as a basic length unit instead of `centi*meter` in `FIELD` units.

Reply:
Thanks for noticing, when doing this I looked up back and on line 320 for surface tension is used `centi*meter`, then I will also change that to feet.

Reply:
>  320 for surface tension is used centi*meter, then I will also change that to feet.

Please don't.  Surface tension is dyne/cm in all unit systems.

Reply:
Ok, I have added tests for the unit conversion as requested, please let me know if further changes are needed before merging this.